### PR TITLE
Questions

### PR DESCRIPTION
**Pull request by koalarick**
A couple of small styling changes, disabling helpful link after first click, adding thumbnails, changes to package.json to allow jest to work.

Note: I needed to get product title into my component, so for now I added another api call to my component which is probably redundant. We should consider what API calls we want to make from API.js at some point.

### CODE REVIEW

File: client/components/Questions/Question.jsx [hunk 1]
```diff
@@ -36,9 +44,9 @@ const Question = ({ product_id, question, refresh }) => {
   };
 
   const markHelpful = () => {
-    API.markQuestionHelpful(question.question_id).then(() =>
-      refresh(product_id)
-    );
+    API.markQuestionHelpful(question.question_id)
+      .then(() => setMarkedHelpful(true))
+      .then(() => refresh(product_id));
```
Comment (acerslee) [inline client/components/Questions/Question.jsx:49]: For error-handling sake, I would include a catch block here.

File: client/components/Questions/Answer.jsx [hunk 2]
```diff
@@ -10,19 +10,34 @@ const useStyles = makeStyles((theme) => ({
...
+  const isSeller = answer.answerer_name.toLowerCase() === "seller";
+  const [markedHelpful, setMarkedHelpful] = useState(false);
+  const dateOptions = { month: "long", day: "numeric", year: "numeric" };
+
   const markHelpful = () => {
-    API.markAnswerHelpful(answer.id).then(() => refresh(product_id));
+    API.markAnswerHelpful(answer.id)
+      .then(() => setMarkedHelpful(true))
+      .then(() => refresh(product_id));
   };
```
Comment (acerslee) [inline client/components/Questions/Answer.jsx:35]: For error handling sake, I would include a catch block here

**Review by acerslee — APPROVED** [review]
Looks good. My only comment is I would include the .catch blocks in the spots I've made note of.

**Comment by koalarick** [comment]
> Looks good. My only comment is I would include the .catch blocks in the spots I've made note of.

Thanks! Committed those changes to address.

**Review by bmh0013 — APPROVED** [review]
good to go

**Pull request merged**